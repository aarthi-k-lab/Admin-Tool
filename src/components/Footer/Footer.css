.footer {
  font-size: 0.75rem;
  color: #666;
  align-items: center;
  padding: 0.25rem;
  justify-content: center;
  text-align: center;
}
