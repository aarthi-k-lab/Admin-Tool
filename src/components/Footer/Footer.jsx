import React from 'react';
import './Footer.css';

const Footer = () => (
  <footer styleName="footer">
    Â© Nationstar Mortgage LLC, All rights reserved
  </footer>
);

export default Footer;
