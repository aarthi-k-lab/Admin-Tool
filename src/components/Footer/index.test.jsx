import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '.';

describe('<Footer />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
