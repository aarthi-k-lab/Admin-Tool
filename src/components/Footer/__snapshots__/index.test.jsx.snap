// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> snapshot test 1`] = `
<footer
  className="components-Footer-Footer__footer--2oWe5"
>
  Â© Nationstar Mortgage LLC, All rights reserved
</footer>
`;
